src/resources/html-attributes.ts

```diff
@@ -0,0 +1,17 @@
+// this function will ensure that we propertly treat a potential string value for a boolean attribute
+// as the boolean representation
+export function normalizeBooleanAttribute(attributeName: string, value: boolean | string): boolean {
```

Comment:
I just feel if we are going to have the comment there we should make the most of it.